Silicon/MeteorlakePkg/Library/BootGuardLibCBnT/BootGuardTpmEventLogLib.c

```diff
@@ -1018,6 +1041,182 @@ CreatePolicyDataMeasurementEvent (
   return Status;
 }
 
+/**
+  Calculate Authority PCR extend value.
+
+  @param[in]  ActivePcrBanks  Active PCR banks in TPM
+  @param[out] Sha256Digest    SHA 256 digest calculation for authority measure
+  @param[out] Sha384Digest    SHA 384 digest calculation for authority measure
+  @param[out] Sm3Digest       SM3 digest calculation for authority measure
+**/
+BOOLEAN
+CaculateAuthorityPCRExtendValue (
+  IN UINT32  ActivePcrBanks,
+  OUT UINT8 *Sha256Digest,
+  OUT UINT8 *Sha384Digest,
+  OUT UINT8 *Sm3Digest
+  )
+{
+  ACM_HEADER_4                             *Acm;
+  KEY_MANIFEST_STRUCTURE                   *Km;
+  BOOT_POLICY_MANIFEST_HEADER              *Bpm;
+  MAX_AUTHORITY_PCR_DATA                   MaxAuthorityPcrData;
+  UINT8                                    *AuthorityPcrDataPtr;
+  UINT32                                   AuthorityPcrDataSize;
+  UINT8                                    *CurrPos;
+  UINT16                                   KeyModulusSize;
+  UINT8                                    MaxModulusExpo[RSA_KEY_SIZE_3K/8 + 4];
+  UINT16                                   Index;
+
+  Acm = FindAcm ();
+  ASSERT (Acm != NULL);
+  if (Acm == NULL) return FALSE;
```

Comment:
Following coding style, it could be changed to:
if (Acm == NULL) {
  return FALSE;
}
ASSERT could be removed since there is a check in line 1073. you could add debug info message if you want to capture error message. Same comments to similar code in this function.